cmd/oras/root/cp.go

```diff
@@ -117,7 +118,9 @@ func runCopy(ctx context.Context, opts copyOptions) error {
 	committed := &sync.Map{}
 	extendedCopyOptions := oras.DefaultExtendedCopyOptions
 	extendedCopyOptions.Concurrency = opts.concurrency
-	extendedCopyOptions.FindPredecessors = graph.FindReferrerPredecessors
+	extendedCopyOptions.FindPredecessors = func(ctx context.Context, src content.ReadOnlyGraphStorage, desc ocispec.Descriptor) ([]ocispec.Descriptor, error) {
+		return graph.Referrers(ctx, src, desc, "")
+	}
```

Comment:
`graph.Referrers` functions differently with `graph.FindReferrerPredecessors`. They are not interchangeable.

Reply:
Let's have a discussion on this.

Reply:
> Let's have a discussion on this.

Online or offline?

Reply:
`graph.Referrers` functions as a superset of with `graph.FindReferrerPredecessors`

Reply:
After an offline discussion, the `Referrers()` can be used for `FindPredecessors`.